src/lib/y2firstboot/clients/licenses.rb

```diff
@@ -69,18 +72,35 @@ def refusal_action
 
       # Directories in which look for the license agreement texts
       #
-      # NOTE: if that result in an empty list, {Yast::InstLicenseClient} will do
-      # an extra attemp to look for the license agreement in the path given as
-      # "base_product_license_directory" global param through the control file.
+      # NOTE: if that result in an empty list and the "directory" argument was ommited too,
+      # {Yast::InstLicenseClient} will do an extra attemp to look for the license agreement in the
+      # path given as "base_product_license_directory" global param through the control file.
+      #
+      # @return [Array<String>] license agreement paths
+      def directories
+        (module_license_directories + sysconfig_licenses_directories).compact.reject(&:empty?).uniq
+      end
+
+      # Directories defined by the module arguments
+      #
+      # NOTE: for the time being, it is only based in the "directory" argument, but it should be
+      # used also if the support for the "directories" argument is implemented.
       #
       # @return [Array<String>] license agreement paths
-      def licenses_directories
-        directories = [
+      def module_license_directories
+        [
+          args["directory"]
```

Comment:
I would return an empty list if no directory is given:

```suggestion
          [ args["directory"] ].map(&:to_s).reject(&:empty?)
```